.github/workflows/build-deploy-pudl.yml

```diff
@@ -26,6 +26,14 @@ jobs:
         if: ${{ (github.event_name == 'schedule') }}
         run: |
           echo "This action was triggered by a schedule." && echo "GCE_INSTANCE=pudl-deployment-dev" >> $GITHUB_ENV && echo "GITHUB_REF=dev" >> $GITHUB_ENV
+          echo "RUN_DATE=$(date +%Y-%m-%d)" >> $GITHUB_ENV
+
+      - name: Tag nightly build
+        if: ${{ (github.event_name == 'schedule') }}
+        uses: EndBug/latest-tag@1
```

Comment:
Do you think we would be able to use `run: git tag | git push <tag>` here instead of a third-party action? Or are there weird issues with that?

Reply:
What does piping the output of `git tag` to `git push` do?

I imagine there's some way we can run `git tag nightly-2023-12-20` and `git push nightly-2023-12-20` but I guess I don't know whether there's other authentication or git configuration magic that has to be done to make that work. Who is the user that's running these commands and what permissions do they have on the repository and how do you communicate that authentication information to the shell that's being used to run the commands? I guess I can try with a test tag and then delete it.

Reply:
It does like that: 
```
Committer identity unknown
*** Please tell me who you are.
Run
  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
to set your account's default identity.
```